src/features/subscription/page/OnboardingSubscription.tsx

```diff
@@ -37,11 +40,28 @@ export const OnboardingSubscription = () => {
     initialSubscribedThemes
   )
 
+  const { mutate: updateProfile, isLoading: isUpdatingProfile } = useUpdateProfileMutation(
+    () => {
+      showSuccessSnackBar({
+        message: 'Tes préférences ont bien été enregistrées',
+        timeout: SNACK_BAR_TIME_OUT,
+      })
+    },
+    () => {
+      showErrorSnackBar({
+        message: 'Une erreur est survenue, veuillez réessayer',
```

Comment:
On tutoie l'utilisateur généralement
```suggestion
        message: 'Une erreur est survenue, tu peux réessayer plus tard.',
```

Reply:
On a plusieurs cas de ce wording dans l'applicaiton, est-ce qu'il ne faudrait pas le faire remonter au design ?

Reply:
Je pense que si